DataFormats/Detectors/TRD/src/Tracklet64.cxx

```diff
@@ -43,6 +43,35 @@ GPUd() int Tracklet64::getPadCol() const
   return CAMath::Nint(6.f + mcmCol * ((float)NCOLMCM) + padLocal);
 }
 
+GPUd() float Tracklet64::PadPositionMCM() const
+{
+  return 12.0 - (getPositionBinSigned() * GRANULARITYTRKLPOS);
+}
+
+GPUd() float Tracklet64::PadPosition() const
```

Comment:
isn't this a duplicate of `getPadCol()`?

Reply:
> isn't this a duplicate of `getPadCol()`?

getPadCol() returns an int, but the position of a tracklet is known better than a pad. Therefore I think it makes sense to provide a function that gives the tracklet position with float precision.
I suspect changing getPadCol() to return a float might break some code, hence I introduced another helper function. I am thinking about calling it getPadColF() (or getPadColFloat, suggestions welcome).